test/integration_test/utils/px_operator.go

```diff
@@ -71,7 +71,18 @@ func getPXOperatorImageTag() (string, error) {
 					}
 				}
 			} else {
-				logrus.Infof("Get portworx-operator image installed [%s]", container.Image)
+				var isMarketplaceDeployment bool
+				for _, env := range container.Env {
+					if env.Name == "OPERATOR_CONDITION_NAME" {
+						isMarketplaceDeployment = true
+					}
+				}
+				if isMarketplaceDeployment && strings.Contains(container.Image, "-dev") {
+					logrus.Infof("Looks like portworx-operator was installed via Openshift Marketplace, with unreleased image [%s]", container.Image)
+					tag = strings.Split(container.Image, ":")[1]
+					return tag, nil
+				}
+				logrus.Infof("Looks like portworx-operator was not installed via Openshift Marketplace, will use tag from the container image [%s]", container.Image)
```

Comment:
can we print the OPERATOR_CONDITION_NAME env value as well? will be easier to debug if the value is not expected.

Reply:
Done